grammars/pegcoffee.cson

```diff
@@ -62,8 +62,24 @@
 'repository':
   'charSet':
     'begin': '\\['
-    'end': '\\]'
+    'beginCaptures':
+      '0':
+        'name': 'keyword.operator'
+    'end': '\\](i?)'
+    'endCaptures':
+      '0':
```

Comment:
Is the intent here to match the whole `]i`, or just the `]`? if the latter, you should capture the closing bracket and use the first capture instead.

Reply:
I missed your comment. Should be addressed now